easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.3.2-foss-2016a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'OSU-Micro-Benchmarks'
+version = '5.3.2'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC=$MPICC CXX=$MPICC'
```

Comment:
@RvDijk please wrap these in double quotes, since the `$MPICC` value may contain spaces...

```python
configopts = 'CC="$MPICC" CXX="$MPICC"'
```